en/ros/ros1_via_ros2.md

```diff
@@ -0,0 +1,445 @@
+# ROS (1) via ROS 2 Bridge (User Guide)
+
+UNDER CONSTRUCTION
```

Comment:
Ok good that it is because basically we just moved all that was im ros2_comm to here and that's not the goal.

Reply:
yeah, we are pretty much just going to leave this open as is, and not really invest any resources into this area.

My view is that this is only viable because open robotics made the bridge happen, if their stuff breaks, we are left on our own, we should avoid that, plus, the only reason you would want this is if you really really wanted to use ROS 1 with microRTPS, instead of using MAVROS, I don't know how many use cases or users we have in that situation, my hope is 0.

Reply:
I get that but we basically moved all the instructions regarding the ROS 2 part as well here. Or am I seeing wrong?

Reply:
@TSC21 This bit has completely changed. If you wait 20 mins you'll see it rendered in https://docs.px4.io/v1.test_r2/en/ros/ros2_comm.html

I actually like these docs now, in that I think I understand what I need to do.